config/locales/en.yml

```diff
@@ -734,6 +734,7 @@ en:
       - Article
       - Unit
       - Unit quantity
+      - Netto price
```

Comment:
if you compare it to the other translations "Nettopreis" in german is usually "Price" in english

please also update the other languages. the translation should be easy to find